app/Models/TeamSpeak/Registration.php

```diff
@@ -59,8 +60,16 @@ public function delete($tscon = null)
             if ($tscon == null) {
                 $tscon = TeamSpeak::run('VATSIM UK Registrations');
             }
+
             if ($this->confirmation) {
-                $tscon->privilegeKeyDelete($this->confirmation->privilege_key);
+                try {
+                    $tscon->privilegeKeyDelete($this->confirmation->privilege_key);
+                } catch (TeamSpeak3_Adapter_ServerQuery_Exception $e) {
+                    if ($e->getMessage() === 'ok') {
```

Comment:
Possibly worth adding a comment above here as per the description in the PR, if we ever come to this again and go 'wtf'

Reply:
Git blame will help us find this... Although I hope I never have to touch it again 🙃